src/resources/assets/js/components/enso/bulma/VueFilter.vue

```diff
@@ -28,7 +28,7 @@
                             :class="value === null ? 'has-text-danger' : 'has-text-success'">
                             <fa icon="power-off"></fa>
                         </span>
-                        <span class="filter-label">{{ __(offLabel) }}</span>
+                        <span class="filter-label" v-if="!icons">{{ offLabel }}</span>
                     </a>
                 </li>
             </ul>
```

Comment:
Please cleanup (remove) the lines responsible with the `__` helper / `mapGetters` (45 and 89-91)